R/mod_packageDependencies.R

```diff
@@ -123,7 +124,13 @@ packageDependenciesServer <- function(id, selected_pkg, user, credentials, paren
         }
       )
       # this is so the dependencies is also a 0x2 tibble like suggests
-      if (rlang::is_empty(pkgref()$dependencies[[1]])) depends(dplyr::tibble(package = character(0), type = character(0), name = character(0)))
+      if (rlang::is_empty(pkgref()$dependencies[[1]])) {
```

Comment:
Meant to comment on this earlier, but these lines seem completely unnecessary base on the `tryCatch()`'s above

Reply:
Ironically, this change is what actually prevents the app from crashing. But you're right. It is redundant since the tryCatch handles it. I removed it with a recent commit.